sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayoutController.java

```diff
@@ -261,6 +289,21 @@ private WritableMap getDiscovery() {
       return discoveryresults;
   }
 
+  private void getClosedCaptionView() {
```

Comment:
misnamed. this is more like "updateAvailableClosedCaptionsLanguages.

Also, how does this even work??? when do you even pass the languages up??